app/views/devise/registrations/edit.html.slim

```diff
@@ -0,0 +1,69 @@
+h2
+  | Edit 
```

Comment:
You can write it in one row
h2 Edit.
BTW, does it stylized with your bootstrap template?, Seems that it doesn't.